easybuild/easyconfigs/d/dxpy/dxpy-0.345.0-foss-2022b.eb

```diff
@@ -0,0 +1,80 @@
+# Updated to latest version
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'dxpy'
+version = '0.345.0'
+
+homepage = 'http://autodoc.dnanexus.com/'
+description = "DNAnexus Platform API bindings for Python"
+
+toolchain = {'name': 'foss', 'version': '2022b'}
```

Comment:
This doesn't seem to use things like BLAS and MPI, so I guess we can lower the toolchain to GCCcore?

Reply:
Done.